### PR TITLE
tests: Replace kubevirt client getter with one that has no error

### PR DESCRIPTION
**Pull request by EdDev**
**What this PR does / why we need it**:

In the end-to-end tests the kubevirt client is frequently fetched and used. The client getter implements a singleton which creates a client once and afterwards returns the same repeatedly.

As such, an error may occur only once, at the initial creation. Any call afterwards will just return the same, which in practice means that there will be no such scenario (first usage is in the test suite setup, which will fail all the suite immediately).

Therefore, there is no point in repeatedly checking the error at each usage.
This change wraps the kubevirt client getter with a function that does not return the error. Instead, it just panics if there is an error. The existing convention already raised a panic on any error received, therefore there is no significant logic change in most usages.

New code should use this client wrapper to simplify the code. Furthermore, it is recommended to call the getter directly instead of passing the client itself down the stack.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:
```release-note
NONE
```

### CODE REVIEW

**Comment by EdDev** [comment]
change: Rebase

**Comment by EdDev** [comment]
change: Rebase

**Comment by EdDev** [comment]
We seem to have an odd behavior with `make generate`.
Many files result with this change:
```
-Copyright 2022 The KubeVirt Authors.
+Copyright 2023 The KubeVirt Authors.
```

Most files are under the `staging` folder.

**Comment by enp0s3** [comment]
> We seem to have an odd behavior with `make generate`. Many files result with this change:
> 
> ```
> -Copyright 2022 The KubeVirt Authors.
> +Copyright 2023 The KubeVirt Authors.
> ```
> 
> Most files are under the `staging` folder.

@EdDev Hi, for now we will update the changes with this [PR](https://github.com/kubevirt/kubevirt/pull/9005), I will investigate why and how we can improve it.

**Review by enp0s3 — COMMENTED** [review]
@EdDev Thank you!
/approve

**Comment by EdDev** [comment]
change: Rebase

File: tests/framework/kubevirt/clientset.go [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+// any repeating call would just repeat the same results.
+//
+// In the context of tests, it is enough to declare panic,
+// declaring that there is a problem in the test suite.
+// This wrapper is in place to avoid the need to repeat the
+// handling of the error each time.
+func Client() kubecli.KubevirtClient {
+	client, err := kubecli.GetKubevirtClient()
+	if err != nil {
+		panic(err)
```
Comment (nunnatsa) [inline tests/framework/kubevirt/clientset.go:37]: I wonder if we need to add some text here to help the user to understand what went wrong.

e.g.
```suggestion
		panic(fmt.Errorf("failed to create the KubeVirt client; %w", err))
```

Comment (EdDev) [reply tests/framework/kubevirt/clientset.go:37]: I would expect the information to be included in the error itself.
This is also the current state today.

**Pull request merged**